projects/packages/scheduled-updates/src/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-update-schedules.php

```diff
@@ -541,18 +541,7 @@ public function prepare_response_for_collection( $response ) {
 	 */
 	public function prepare_item_for_response( $item, $request ) {
 		$item = (array) $item;
-
-		$status = Scheduled_Updates::get_scheduled_update_status( $item['schedule_id'] );
-		if ( ! $status ) {
-			$status = array(
-				'last_run_timestamp' => null,
-				'last_run_status'    => null,
-			);
-		}
-
-		$item                       = array_merge( $item, $status );
-		$item['health_check_paths'] = Scheduled_Updates_Health_Paths::get( $item['schedule_id'] );
-
+		$item = apply_filters( 'jetpack_scheduled_response_item', $item, $request );
```

Comment:
I agree, setting these values in `get_items` directly, while we rely on actions in `create_item`, `update_item`, and `delete_item` is odd. It's driven by the various data sources we have for all of these values.

At some point we used `register_rest_field()` to add these values from within this endpoint, which felt odd. I wonder if it would feel less odd, if we did that from the individual files instead, `Scheduled_Updates_Health_Paths`, `Scheduled_Updates_Logs`, etc? 

Because essentially this new filter does the same thing as `add_additional_fields_to_object()` on the next line.